src/navigation/navigation.tsx

```diff
@@ -527,12 +544,17 @@ const AuthenticationStack = (): JSX.Element => {
       <Stack.Screen
         name={ScreenRoutesEnum.LOCK}
         component={SSILockScreen}
-        options={{
-          headerTitle: translate('lock_title'),
+        initialParams={{onVerificationSuccess: onLogin}}
+        options={({route}) => ({
+          headerTitle: route.params.headerTitle ? route.params.headerTitle : translate('lock_title'),
           header: (props: NativeStackHeaderProps) => (
-            <SSIHeaderBar {...props} showBackButton={false} showProfileIcon={false} headerSubTitle={translate('lock_subtitle')} />
+            <SSIHeaderBar
+              {...props}
+              //showBackButton={Platform.OS === PlatformsEnum.IOS}
+              headerSubTitle={route.params.headerSubTitle ? route.params.headerSubTitle : translate('lock_subtitle')}
```

Comment:
since we stepped away from the reusing the same stack, we can set only translate('lock_subtitle') as subtitle as this stack is only used for login

Reply:
chaged